src/renderer/react/app/nodes/NodeManager.messages.js

```diff
@@ -0,0 +1,6 @@
+import { defineMessages } from 'react-intl';
```

Comment:
If you write it like this (`ComponentName.messages.js`) we need to formalize it bc elsewhere we write it `component-name.messages.js`. I don't have a preference but with your approach at least the files are guaranteed to be next to each other.

Reply:
What's the convention for `.module.scss` files ? I'm in favor of refactoring everything in the convention `ComponentName.*`, what do you think ?

Reply:
I'm good with that, @maxence-lefebvre ?

Reply:
will do that in another PR